common/v2/features/Settings/components/GeneralSettings.tsx

```diff
@@ -94,31 +102,48 @@ export default class GeneralSettings extends React.Component<SettingsProps> {
     }
   };
 
+  public renderTooltipContent = () => <div>Test</div>;
+
   public render() {
     const { globalSettings } = this.props;
     return (
-      <DashboardPanel heading="General Settings">
+      <DashboardPanel heading={translateRaw('SETTINGS_GENERAL_LABEL')}>
```

Comment:
Any reason why you're using `translateRaw` instead of `translate`?
Applies to the entire component

Reply:
I think you missed this line. Please change to `translate`

Reply:
Oh. Didn't miss, just thought that `heading` could only be a string. Looks like it can be `any` though so I will change it!